.goreleaser.yml

```diff
@@ -58,6 +58,14 @@ brews:
     license: MIT
     test: |
       system "#{bin}/s5cmd version"
+      
+nfpms:
+  -
+    vendor: Peak
+    homepage:  https://github.com/peak/s5cmd
+    license: MIT
```

Comment:
You could add yourself as the maintainer if you want with something like;
```yaml
    maintainer: name@host <Name>
```

Reply:
I'm not sure I should be the listed maintainer, but also it looks like the maintainer field will be mandatory soon: https://goreleaser.com/deprecations/

Do you have in mind a specific value, or it could just be something like `Peak <contact@peak.com>` (hopefully no one will actually try to contact someone this way)?

Reply:
Let's go with `Peak <packaging@peak.com>` then.